vizro-ai/examples/dashboard_ui/requirements.in

```diff
@@ -3,3 +3,5 @@ vizro-ai>=0.2.1
 black
 jupyter
```

Comment:
Remind me, why do we need jupyter again?

Reply:
Hm.. I'm not really sure. I cannot see why we would need it, tbh. Should I remove it?

Reply:
Yes, also does `vizro-ai` not need to `>=0.3.0`?